api/authFunctions.ts

```diff
@@ -0,0 +1,23 @@
+import { account } from "./config"
+// import { useRouter } from 'next/navigation';
+import { NextRouter } from 'next/router';
+
+
+  export const login = async (router: NextRouter, email: string, password: string) => {
+    try {
+      await account.createEmailPasswordSession(email, password);
+      router.push('/');
+    } catch (error) {
+      console.log(error);
```

Comment:
I'm not sure why this was reverted to log instead of error. This should be console.error() for any errors.